test/scf_compare.jl

```diff
@@ -27,4 +27,11 @@ include("testcases.jl")
         ρ_alg = fourier(scfres.ρ)
         @test maximum(abs.(ρ_alg - ρ_nl)) < 30tol
     end
+
+    # Run other mixing with nlsolve (the others are too slow...)
+    for mixing in (KerkerMixing(), SimpleMixing(), nothing)
+        scfres = self_consistent_field!(Hamiltonian(basis, ρ0), n_bands, tol=tol, solver=scf_nlsolve_solver(), mixing=mixing)
```

Comment:
Pretty long line.

Reply:
It's a test :-p

Reply:
Sure, but I'll still need to read that code from time to time. :-p

Reply:
fix your editor ;-)
(I broke the line)

Reply:
I am happy with 3 letters, 5 is harder to type (and aliases are cheating!) :smile:

Reply:
Ah, those weird people who type the command name of their editor. Can't remember the last time I did that: I just run emacs when my computer boots and do everything from there.

Reply:
Sure, I mean that's why it's not an editor but an operating system :P